src/css/_population-slider.scss

```diff
@@ -35,7 +35,7 @@
     font-size: 0.5em;
     opacity: 1;
     flex: 1;
-    text-align: center;
+    text-align: left;
```

Comment:
Necessary for the label to line up properly with the slider.

Reply:
Can we add one more level from 50-100? I think that granularity is more important for the smaller cities.